gridsome.server.js

```diff
@@ -36,6 +38,31 @@ class VueI18n {
    * @param {function} param.removePage
    */
   createManagedPages({ findPages, createPage, removePage }) {
+    
+    // Create custom localized routes using the this.options.routes object
+    if (this.options.routes) {
```

Comment:
or directly adding the check in the first if:
```suggestion
    if (this.options.enablePathGeneration === false && this.options.routes) {
```
to not mix page and route generation. Thinking about an user that test the functionality setting routes parameters but want to disable without removing the configuration.